### PR TITLE
New FDD LTE bands

### PR DESCRIPTION
**Pull request by lfarizav**
New FDD LTE bands
Signed-off-by: Luis Felipe Ariza Vesga <lfarizav@unal.edu.co>

## Summary
This pull request is required to configure eNBs at the NMS for FDD LTE bands from 1-28 and 71. This PR will impact FreedomFi NMS orchestration service and the NMS service build on-premise for everyone who wants to create PoCs in those bands mentioned before. At this moment only supports FDD band 1-4 and 28 and TDD band 33-52.

## Test Plan
 I will test both, on-premise and with FreedomFi NMS service. Also. I will use openairinterface eNB and USRP b210/N310 to transmit signals over the air.

## Additional Information

- [ ] This change is backwards-breaking

> Please fill up the summary (why this PR is needed, what is its impact) and test plan (how it is tested/used).
This PR is required to test more FDD LTE bands. Nowadays, there are a few of them. In the future, all bands should be added.

### CODE REVIEW

**Review by hcgatewood — COMMENTED** [review]
Adding @andreilee and @ulaskozat as they're likely most familiar

**Comment by ulaskozat** [comment]
Please fill up the summary (why this PR is needed, what is its impact) and test plan (how it is tested/used).

**Comment by lfarizav** [comment]
> Please fill up the summary (why this PR is needed, what is its impact) and test plan (how it is tested/used).

Dear @ulaskozat. I have already completed the summary and test plan.

**Comment by andreilee** [comment]
@lfarizav Has this been replaced with another PR? Why did you close this PR?

**Comment by lfarizav** [comment]
> @lfarizav Has this been replaced with another PR? Why did you close this PR?

@andreilee I am not sure what happened. I do not know if new FDD bands for LTE will be merged into the master branch using another PR. By the way, we can do the changes by ourselves and enable the bands we need.

**Comment by andreilee** [comment]
Reopening this PR

File: lte/cloud/go/lte/lte_bands.go [hunk 1]
```diff
@@ -40,7 +40,29 @@ var bands = [...]LTEBand{
...
+        {ID: 19, Mode: FDDMode, StartEarfcnDl: 6000, StartEarfcnUl: 24000, CountEarfcn: 150},
+        {ID: 20, Mode: FDDMode, StartEarfcnDl: 6150, StartEarfcnUl: 24150, CountEarfcn: 300},
+        {ID: 21, Mode: FDDMode, StartEarfcnDl: 6450, StartEarfcnUl: 24450, CountEarfcn: 150},
+        {ID: 22, Mode: FDDMode, StartEarfcnDl: 6600, StartEarfcnUl: 24600, CountEarfcn: 900},
+        {ID: 23, Mode: FDDMode, StartEarfcnDl: 7500, StartEarfcnUl: 25500, CountEarfcn: 200},
+	{ID: 24, Mode: FDDMode, StartEarfcnDl: 7700, StartEarfcnUl: 25700, CountEarfcn: 340},
+        {ID: 25, Mode: FDDMode, StartEarfcnDl: 8040, StartEarfcnUl: 26040, CountEarfcn: 650},
+        {ID: 26, Mode: FDDMode, StartEarfcnDl: 8690, StartEarfcnUl: 26690, CountEarfcn: 350},
+        {ID: 27, Mode: FDDMode, StartEarfcnDl: 9040, StartEarfcnUl: 27040, CountEarfcn: 170},
+	{ID: 28, Mode: FDDMode, StartEarfcnDl: 9210, StartEarfcnUl: 27210, CountEarfcn: 59376},
```
Comment (ulaskozat) [inline lte/cloud/go/lte/lte_bands.go]: Why is `CountEarfcn` is changed to 59376 from 450 for ID 28?

File: lte/cloud/go/lte/lte_bands.go [hunk 2]
```diff
@@ -40,7 +40,29 @@ var bands = [...]LTEBand{
...
+	{ID: 71, Mode: FDDMode, StartEarfcnDl: 68586, StartEarfcnUl: 133122, CountEarfcn: 59426},
```
Comment (ulaskozat) [inline lte/cloud/go/lte/lte_bands.go]: For ID: 71, `CountEarfcn` should be 350.

**Review by ulaskozat — CHANGES_REQUESTED** [review]
Please check the additional comments on earfcn counts.

File: lte/cloud/go/lte/lte_bands.go (hunk 1 above)
Comment (lfarizav) [reply lte/cloud/go/lte/lte_bands.go]: Dear @ulaskozat the correct CountEartfcn values were changed.

Comment (lfarizav) [reply lte/cloud/go/lte/lte_bands.go]: > Why is `CountEarfcn` is changed to 59376 from 450 for ID 28?

Sorry, it was a mistake, values were updated.

**Review by ulaskozat — APPROVED** [review]
LGTM

**Review by lfarizav — COMMENTED** [review]
I correct the mistake of the CountEarfcn for bands 28 and 71. I apologize because of the mistake. The changes are the following: 
	{ID: 28, Mode: FDDMode, StartEarfcnDl: 9210, StartEarfcnUl: 27210, CountEarfcn: 450},
	{ID: 71, Mode: FDDMode, StartEarfcnDl: 68586, StartEarfcnUl: 133122, CountEarfcn: 350},

**Comment by lfarizav** [comment]
> @lfarizav Has this been replaced with another PR? Why did you close this PR?

Sorry @andreilee, I did not close the PR. It was done automatically.

**Comment by hcgatewood** [comment]
Closing due to inactivity. Please reopen if desired @lfarizav

**Comment by ulaskozat** [comment]
@hcgatewood why is this closed. The inactivity looks on the side of code owners and not the PR owner.

**Comment by hcgatewood** [comment]
> @hcgatewood why is this closed. The inactivity looks on the side of code owners and not the PR owner.

My impression was "last update May 11", and some CI checks were failing. I'm fine with reopening if desired 🙂

**Comment by ulaskozat** [comment]
> > @hcgatewood why is this closed. The inactivity looks on the side of code owners and not the PR owner.
> 
> My impression was "last update May 11", and some CI checks were failing. I'm fine with reopening if desired 🙂

which checks were failing? @lfarizav let us know if you want this to be reopened and merged. Could you also post your test results?

**Pull request closed without merging**